contracts/CO30_Andrew_HotInHerre.ruha/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,26 @@
+Ruha has seen inner turmoil. We've received a secondary contract to assist them in their time of need.
+
+<font color='#e3d310'>EMPLOYER</font>
+Ruha Government
+The Ruha Government is desperate for assistance. They're begging for any help as they have no one to assist their militia and law enforcement agency.
+
+<font color='#d81717'>ENEMY</font>
+Looters & Rioters
+Known to be anti-government civilians, they are defined as enemy. Armed and possibly dangerous, they will not give a second thought to kill anyone in their way for their gain. Dressed like a normal civilian, these anti-government individuals will be armed with basic firearms and use guerrilla tactics.
+
+Unknown Cult faction
+A cult like organization has stepped up to the plate to fill the void in the power vacuum. They have been seen wearing tinfoils hats, and can range to being armed as much as a farmer to being geared to the nine. These individuals will die for their cause, believing it to be for the common good. Known to possible have access to armed technicals and might have commondere an apc or two from the local forces. Be weary. 
+
+<font color='#1d69f6'>FRIENDLY</font>
+Ruha Law Enforcement
+Locations: Metsälä Airport, Ruha Police HQ, and patrolling throughout the region.
+
+Ruha Pro-Government Militia
+Locations: Kivimäki, Länsikylä Sawmill, and patrolling throughout the region.
+
+<font color='#993399'>CIVILIAN</font>
+IDAP
+Location: Kaaranmännikkö Stadium
+IDAP is trying to provide well needed supplies to civilians in need. They've set up a camp in the stadium and are doing their best to provide those who are in need. It has been reported that there has been trouble in that area with Anti-Government members stirring trouble. Often they steal or rob supply trucks bringing supplies in. There is not enough Pro-Government forces to be there 24/7. Be mindful when approaching.
```

Comment:
```suggestion
IDAP is trying to provide well needed supplies to civilians in need. They've set up a camp in the stadium and are doing their best to provide those who are in need. There is not enough Pro-Government forces to be there 24/7. Be mindful when approaching.
```
Only info about IDAP should go under IDAP, Info about other factions and their activities should go in mission or under that faction's section